my-app/src/Pages/CreateMovie.tsx

```diff
@@ -0,0 +1,155 @@
+import React from "react";
+import { withFormik, FormikProps } from "formik";
+import { movieCreateSchemas } from "../Schemas/movieCreateSchemas";
+import { movieService } from "../Services/MovieService";
+import useAuth from "../Hooks/useAuth";
+import { Button } from "react-bootstrap";
+import { Form } from "react-bootstrap";
+
+interface FormValues {
+  title?: string;
+  description: string | undefined;
+  url: string | undefined;
+  genre: Array<string> | undefined;
+}
```

Comment:
Use one way of defining a variable as undefined, not multiple

```suggestion
  title: string | undefined;
  description: string | undefined;
  url: string | undefined;
  genre: Array<string> | undefined;
}
```